indico/modules/events/abstracts/templates/abstract.html

```diff
@@ -20,12 +21,19 @@
     {{ render_abstract_public(abstract) }}
     <div id="abstract-private" class="i-timeline">
         {{ render_timeline(abstract, comment_form, review_form) }}
+        {% if abstract.can_judge(session.user, check_state=true) and abstract.event_new.tracks %}
+            {{ render_reviewed_for_tracks_box(abstract, review_track_list_form) }}
+        {% endif %}
+
+        {% if abstract.can_see_reviews(session.user) %}
+            <div id="abstract-review-summary-box" class="i-timeline-item">
+                {{ render_review_summary_box(abstract, judgment_form) }}
```

Comment:
BTW, should we even show the by-question summary in this case? And if yes, do we want to restrict the avg there to tracks you can see reviews for?